honestcash-v2/src/app/main/components/story-actions/story-actions.component.html

```diff
@@ -0,0 +1,4 @@
+<div class="col">
```

Comment:
refactoring is to be done like style into scss file. these components that start with main-story- are story related components. I will probably abstract story into its own module aswell inside main. we'll see